### ISSUE TITLE
Change text color

### ISSUE THREAD

**Issue by wpalchemist** [RagtagOpen/freefrom-website#68]
Throughout the site, anywhere there is white text on a colored background, we need to change the text color to black for better accessibility.

There are two steps to this process: we need to update the colors in all existing blocks on the site, and we also need to update the block patterns to use black.
